realtime/README.md

```diff
@@ -1,17 +0,0 @@
-## Node.js realtime server
```

Comment:
we don't really need one, since the realtime server is served through the same server as this whole js repo, we could make some comments in the readme about it